internal/driver/glfw/canvas.go

```diff
@@ -381,7 +386,8 @@ func (c *glCanvas) ensureMinSize() bool {
 	shouldResize := windowNeedsMinSizeUpdate && (c.size.Width < min.Width || c.size.Height < min.Height)
 	c.RUnlock()
 	if shouldResize {
-		c.Resize(c.Size().Union(c.MinSize()))
+		// We need to resize the Window actually as the content doesn't fit at the moment
+		c.context.(*window).Resize(c.Size().Union(c.MinSize()))
```

Comment:
See my previous comment.

Reply:
In this instance it seems OK - I suppose the code could be moved into window to tidy things up a little.
Context() returning *window is not a co-incidence, it is how this driver is designed. The generic nature is to support passing GL contexts in different drivers. We pass the value in and get it back again so it's pretty safe.